src/components/AssetsDropZone/AssetsDropZone.scss

```diff
@@ -4,19 +4,21 @@
   text-align: center;
 }
 
-.drop-zone {
+.drop-zone-compact {
   outline: $dark dashed 1px;
-  padding: $spacer;
+  padding: 0;
```

Comment:
I think we could use a *touch* more padding here because the dotted line is riding up right next to the icon and max file size text. Maybe `$spacer / 4`?